src/ethereum/eth.js

```diff
@@ -19,13 +19,18 @@ const eth = {
   /**
    * Connect to web3
    * @param  {string} [defaultAccount=web3.eth.accounts[0]] - Override the default account address
-   * @param  {array<Contract>} [contracts] - An array of objects defining contracts or of Contract subclasses to use. By default will use the available contracts.
+   * @param  {array<Contract>} [contracts] - An array of objects defining contracts or of Contract subclasses to use. By default will use the available contracts
+   * @param  {object} [options] - Extra options for the ETH connection
    * @return {boolean} - True if the connection was successful
    */
-  connect(defaultAccount, contracts) {
+  connect(defaultAccount, contracts, options) {
```

Comment:
`options = {}`, otherwise it's an interface change; line `29` will explode if client API does not update accordingly (can't read property `httpProviderUrl` of `undefined`)